actions/class.Items.php

```diff
@@ -126,7 +126,27 @@ public function moveInstance()
     {
         return parent::moveInstance();
     }
-    
+
+    /**
+     * overwrite the parent moveAllInstances to add the requiresRight only in Items
+     * @see tao_actions_TaoModule::moveResource()
+     * @requiresRight uri WRITE
+     */
+    public function moveResource()
+    {
+        return parent::moveResource();
+    }
+
+    /**
+     * overwrite the parent moveAllInstances to add the requiresRight only in Items
+     * @see tao_actions_TaoModule::moveAll()
+     * @requiresRight destinationClassUri WRITE
```

Comment:
The rights should be set on the list of ids, the destination is not selected up front